### ISSUE TITLE
A mathematical and physical proof for Special Relativity

### ISSUE THREAD

**Issue by Will0214** [leios/SoME_Topics#61]
## About the author

I am a passionate viewer of physics videos around the web and I recently obtained my bachelor's degree in mathematics (actuarial sciences option) at Université de Montréal. I am always curious to dive into the great mysteries of the universe, and so I have recently tried to understand the special theory of relativity with much interest.

## Quick Summary

Although there are many videos explaining with utmost precision on how special relativity predicts certain physical phenomena such as time dilation or length contraction, I have seen none who try to find the roots of this theory. With the help of a preprint review paper that I have made about a month ago, I believe that this subject can be easily made into a video or at least images which are much more explanatory than my paper.

Portraying a certain thought experiment and showing mathematical formulas which would link widely known subjects (such as classical atomic electromagnetism) with the hard-to-grasp postulates of special relativity could make a short but understandable guide for anyone.

## Target medium

A short explanatory video (I can do the voiceover, I have experience with adding audio to videos) or a more visual/graphical understanding of my review paper.

### More details

Here is the abstract of my review paper "Special Relativity in Electromagnetic Interactions of Moving Atoms":

_The phenomena explained by the special theory of relativity have predicted precise observations throughout many fields of physics. In this paper I reviewed the two main occurrences of relativity to find various mathematical relations. As time dilation and length contraction are well known facts, the only additional insight that could be learned from them is their origin. By looking deeper into a moving object, some unseen consequences have shone a light on mechanisms previously stated as universal and without physical cause. The electromagnetic interactions of atoms at high velocities bring forth macroscopic changes that coincide perfectly with time dilation and length contraction. These changes explain the special theory of relativity with further clarity._

Here is a link : https://drive.google.com/file/d/1y-S2jrvtZsuGlTZ5vjGvqrU3OyW9tAC5/view?usp=sharing

Basically, the only animations and/or visual augmentations that could be made into an intuitive explanation of special relativity would be the three figures in my paper as well as the light cone described in my first appendix. I think this would be very interesting for those who are trying their hardest to understand Special Relativity, and those animations/visual augmentations could be trivial for some content producers.

### Contact details

My email address is william.fettich@gmail.com. I'll respond within two days!

**Comment by kamiloze2004** [comment]
I will help your proof for Special Relativity will be much more complexing, but simple to understand for normal people.

**Comment by Will0214** [comment]
> I will help your proof for Special Relativity will be much more complexing, but simple to understand for normal people.

I think with a visual representation of the formulas, anyone will at least understand its implications and so from that they will understand how it can affect moving objects!

I might not understand what you're trying to say though, I'm sorry if that's the case...

**Comment by kamiloze2004** [comment]
Combining Personal Reality and Combining Dream Personal Reality, and then opposite is true too.

**Comment by farbowitz** [comment]
Hi Will, I'm also looking at relativity as a topic. https://github.com/leios/SoME_Topics/issues/103
I'd be curious to hear more details about what you would want to go into. Want to set up a meet on Discord?

**Comment by Will0214** [comment]
> Hi Will, I'm also looking at relativity as a topic. #103 I'd be curious to hear more details about what you would want to go into. Want to set up a meet on Discord?

Hi! I'll send you an email with my info in a few minutes - although I won't be available soon as explained in my incoming email. I saw your issue and it's indeed an interesting idea, can't wait to discuss it!
